guides/common/modules/proc_creating-custom-snippets-to-apply-errata.adoc

```diff
@@ -0,0 +1,36 @@
+[id="creating-custom-snippets-to-apply-errata_{context}"]
+= Creating custom snippets to apply errata
+
+Custom snippets allow you to execute custom code when applying errata on hosts.
+You can run code before and/or after applying errata.
+
+.Prerequisites
+* Depending on the used template, multiple custom snippet hooks exist which you can use to include custom snippets.
+Ensure that you check your template first to verify which custom snippets you can use.
```

Comment:
```suggestion
Check your template to verify which custom snippets you can use.
```
I don't think you need "first" because this is a prerequisite.

Reply:
Applied.